src/dmd/backend/el.c

```diff
@@ -2240,6 +2240,7 @@ elem *el_convert(elem *e)
                 break;
             }
 #if 1
+            /* FALL-THROUGH */
```

Comment:
Maybe have this before the `#if 1`

Reply:
I tried this, but `gcc` doesn't seem to like it.

Reply:
Is there a break after the `#endif`?

Reply:
FYI: you can click on "expand" on GitHub diffs, for convenience:

![image](https://user-images.githubusercontent.com/4370550/34250484-427a31d2-e63d-11e7-8491-24eda19b30f7.png)

Reply:
Not on my phone I can't. :-(

Reply:
At the bottom of every GitHub page there's a "Desktop version" link which you can use to switch to the full desktop version when you're browsing through your phone.